sqla_yaml_fixtures.py

```diff
@@ -123,9 +129,16 @@ def load(ModelBase, session, fixture_text):
     # make sure backref attributes are created
     sqlalchemy.orm.configure_mappers()
 
+    # Data should be sequence of entry per mapper name
+    # to enforce that FKs (__key__ entries) are defined first
     data = yaml.load(fixture_text)
+    if not isinstance(data, list):
+        raise ValueError('top level YAML should be sequence (list)')
+
     store = Store()
-    for model_name, instances in data.items():
+    for model_entry in data:
+        # Model entry is dict with one item
+        model_name, instances = model_entry.popitem()
```

Comment:
Propose to also skip zero length model entries. I have a prog that generates a `sqla_fixture` from SQLAlchemy model. If the table is empty it generates just an empty `- Name:`. Ok?